forum/tests/test_views.py

```diff
@@ -1,38 +1,52 @@
 from uuid import uuid4
-
 from django.test import Client, TestCase
 from django.urls import reverse
-
-from ..factories import CommentFactory
+from ..factories import CommentFactory, ForumFactory, PostFactory, PostVoteFactory, CommentVoteFactory
 from user.factories import UserFactory
 
+#  For use throughout tests...may be bad practice..not sure
+tf = ForumFactory()
+tp = PostFactory()
+tpv = PostVoteFactory()
+tcv = CommentVoteFactory()
 
 class ForumDetailViewTestCase(TestCase):
     def test_200(self):
+        url = tf.get_absolute_url
+        client = Client()
+        response = client.get(url)
+        self.assertTrue(response.status_code == 200, "Site does not exist")
+        #  saw your assert code after the fact....is more concise...i guess lol and no error message is needed
         # TODO
-        self.assertTrue(True)
-
 
 class ForumListViewTestCase(TestCase):
     def test_200(self):
-        # TODO
-        self.assertTrue(True)
+        client = Client()
+        response = client.get("/")
+        self.assertTrue(response.status_code == 200, "Site does not exist")
+
 
 
 class ForumUpdateViewTestCase(TestCase):
     def test_200(self):
-        # TODO
         self.assertTrue(True)
 
 
 class PostDetailViewTestCase(TestCase):
     def test_200(self):
+        # test_slug = Forum.objects.first().slug
+        # test_post_id = Post.objects.first().id
+        # test_post_slug = Post.objects.first().id
+        #  <slug:forum_slug>/<uuid:post_id>/<slug:post_slug>/
+        url = "{}/{}/{}/".format(tf.slug, tp.id, tp.slug)
+        client = Client()
+        response = client.get(url)
+        self.assertTrue(response.status_code == 200, "Site does not exist")
         # TODO
-        self.assertTrue(True)
-
 
 class CommentDetailViewTestCase(TestCase):
     def test_404(self):
+        #  Do not understand what is goin on
```

Comment:
This `test_404` is meant to test a request to an invalid/unknown webpage. That's what `fake_comment_id` is -- a fake UUID. If you plug the fake UUID into `reverse` but keep all the other arguments as valid values, the request will return a 404 (Not Found) response. That's what this is meant to test.